docs/howto.rst

```diff
@@ -47,30 +54,27 @@ First, we can see what information ``phys2bids`` reads from the file, and make s
 
 The simplest way of calling ``phys2bids`` is moving to the folder containing the physiological file and typing:
 
-.. code-block:: shell
-
-    cd phys2bids/phys2bids/tests/data/
-    phys2bids -in tutorial_file
-
 ``phys2bids`` will try to get the extension for you.
-However, we’ll use one more argument to have a sneak peak into the content of the file:
+However, we’ll use -info to have a sneak peak into the content of the file and -outdir to direct all the files to the same directory:
 
 .. code-block:: shell
```

Comment:
I think this code block should be before the line "phys2bids will try to get the extension for you"

Reply:
For more context, it says ... "and typing:" on line 55 but this is not followed by a command